command/core.py

```diff
@@ -28,3 +30,15 @@ def handle_app_mention(self, event_data):
                 return 1
             except KeyError:
                 return -1
+
+    def handle_member_join(self, event_data):
```

Comment:
In the future we will want to refactor this into an `Event` class a la the `Command` classes. For now we can go ahead and merge this - this seems like most of the business logic we'll want.